src/include/simeng/arch/ProcessStateChange.hh

```diff
@@ -0,0 +1,25 @@
+#pragma once
```

Comment:
How come this file needs no includes?

Reply:
That is a great question.... perhaps when it is linked with the files that do use it, it has the necessary headers available?

It also seems to be ok in the Jenkin's pipeline, but I can add the headers it should need if preferable?

Reply:
Headers added along with some general code "de-tangling"